### ISSUE TITLE
As an inline element

### ISSUE THREAD

**Issue by vigna** [carrot/share-button#153]
I need to use the share button as an inline element in the bar of a FuelUX repeater. However, no matter what I do, it will show on a separate line. Any suggestion?

**Comment by joshrowley** [comment]
You should be able to target the share button using CSS and setting `display: inline`. If you're doing that and it's still not working, you are probably having issues with setting the right [CSS specificity](http://www.smashingmagazine.com/2007/07/27/css-specificity-things-you-should-know/).

I'm not familiar with FuelUX, but if you're able to provide a [JSFiddle](https://jsfiddle.net/) or something similar with a recreation of the specific issue, I can help you further. Please reopen if you're able to provide this.

**Comment by vigna** [comment]
Well, I even tried

<div class="share-button" style="display: inline"></div>

I doubt you can be more specific than that. The problem is that when you look at the DOM after loading it's

<div class="share-button sharer-0" style="display: block;">

So someone is killing my style indication and stamping another one over it.

**Comment by joshrowley** [comment]
Yes, it looks like share button is inserting that inline, opening this back up.
